api/tests/functional-tests/crud/test_create_delete.py

```diff
@@ -1162,21 +1180,22 @@ def method_to_test(
     pydantic_metrics = crud.get_evaluations(
         db=db, evaluation_ids=[evaluation_id]
     )
+    assert pydantic_metrics[0].metrics is not None
     for m in pydantic_metrics[0].metrics:
         assert isinstance(m, schemas.Metric)
     assert len(pydantic_metrics[0].metrics) == len(metric_ids)
 
     # run again and make sure no new ids were created
     evaluation_id_again, _, _ = method_to_test(label_key="class")
     assert evaluation_id == evaluation_id_again
-    metric_ids_again = [
-        m.id
-        for m in db.scalar(
-            select(models.Evaluation).where(
-                models.Evaluation.id == evaluation_id_again
-            )
-        ).metrics
-    ]
+
+    query = db.scalar(
+        select(models.Evaluation).where(
+            models.Evaluation.id == evaluation_id_again
+        )
+    )
+    assert query is not None
```

Comment:
another case where we have to break up statements to handle optional return types